crates/nostr-sdk/src/relay/pool.rs

```diff
@@ -123,8 +123,8 @@ impl Default for RelayPool {
 impl RelayPool {
     /// Create new `RelayPool`
     pub fn new() -> Self {
-        let (notification_sender, _) = broadcast::channel(64);
-        let (pool_task_sender, pool_task_receiver) = mpsc::channel(64);
+        let (notification_sender, _) = broadcast::channel(1024);
+        let (pool_task_sender, pool_task_receiver) = mpsc::channel(1024);
```

Comment:
Maybe we can increase this limit. 
`1024` is good for `Relay` and `RelayPoolNotifications` channels but the `RelayPoolTask` channel receive all messages from relays and some can be lost if user adds so many. Maybe that channel can be set as `unbounded`? What do you think? Is it a bad idea?

```rust
let (pool_task_sender, pool_task_receiver) = mpsc::unbounded_channel();
```

Reply:
Good point.

The only issue I see is that it can eat up all the available memory if the caller stops consuming:

> Note that the amount of available system memory is an implicit bound to the channel. Using an unbounded channel has the ability of causing the process to run out of memory. In this case, the process will be aborted.
> https://docs.rs/tokio/latest/tokio/sync/mpsc/fn.unbounded_channel.html

Maybe having it configurable would get the best of both worlds?

- `Client::new()` with a large default
- `Client::new_buffered(u16)` with a custom channel size

Reply:
Yes, it's a safer solution

Reply:
Did you implement it with this PR? Otherwise I'll this new method after the merge.

Reply:
No I didn't.

Reply:
> No I didn't.

Sorry, I used the wrong verb. But anyway, I'll implement that new method later.

Reply:
No problem, thanks!